src/server/models/users.model.js

```diff
@@ -8,6 +8,7 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
+  userId: String,
   email: String,
   name: String,
   age: Number,
```

Comment:
@SarahhhC Let's add those fields in the schema even if we don't have that data automatically. So at least the client folks can start rendering it. 

@heartjigi : in case those data fields are empty, we need a graceful way to display the empty fields in the UI. Make sure the client doesn't crash or anything for those cases :)